src/labels/CustomLabels.labels

```diff
@@ -1100,6 +1107,13 @@ If you’re not sure how to resolve these errors, post a message in the Nonprofi
         <shortDescription>addrSeasonalPartial</shortDescription>
         <value>An address must have all or none of its Seasonal fields set.</value>
     </labels>
+    <labels>
+        <fullName>advConfigException</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Config Error in Adv</shortDescription>
+        <value>The Gift Processing package is not config correctly. Please contact your Admin for more details.</value>
```

Comment:
we try to get the correct text from the Doc writer before we merge into master, otherwise it will be missed and forgotten!

Reply:
Yes, I will check with @lianeyee for this one since @juliesnow is not available today.

Reply:
@weizsforg Couple of things. Should we avoid referring to "adv" everywhere, not just in the label text itself? So . . .
<fullName>giftProcessingConfigException</fullName>
<shortDescription>Gift Processing configuration error</shortDescription>

I left a comment in the PR conversation looking for more context about when this error appears. Assuming it's more common for an end user to see the error, I suggest:
<value>Your org is missing required configuration for Gift Processing. Contact your admin for help.</value>